com.github.bitseather.weather.json

```diff
@@ -0,0 +1,168 @@
+                "url": "https://download.gnome.org/sources/libchamplain/0.12/libchamplain-0.12.16.tar.xz",
+                "sha256": "4a7e31cf7889669aebf04631543af64435edd989685159b804911c6005db908d"
+            }]
+        },
+        {
+            "name": "geocode-glib",
+            "sources": [{
+                "type": "archive",
+                "url": "http://ftp.gnome.org/pub/GNOME/sources/geocode-glib/3.24/geocode-glib-3.24.0.tar.xz",
+                "sha256": "19c1fef4fd89eb4bfe6decca45ac45a2eca9bb7933be560ce6c172194840c35e"
+            }]
+        },
+        {
+            "name": "libdbusmenu",
+            "config-opts": [
+                "--with-gtk=3",
+                "--disable-gtk-doc",
+                "--disable-doc",
+                "--disable-dumper"
+            ],
+            "build-options": {
+                "env": {
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+                }
+            },
+            "sources": [{
+                "type": "archive",
+                "url": "https://launchpad.net/libdbusmenu/16.04/16.04.0/+download/libdbusmenu-16.04.0.tar.gz",
+                "sha256": "b9cc4a2acd74509435892823607d966d424bd9ad5d0b00938f27240a1bfa878a"
+            }]
+        },
+        {
+            "name": "libindicator",
+            "config-opts": ["--with-gtk=3", "--disable-tests"],
+            "sources": [{
+                    "type": "archive",
+                    "url": "https://launchpad.net/libindicator/12.10/12.10.1/+download/libindicator-12.10.1.tar.gz",
+                    "sha256": "b2d2e44c10313d5c9cd60db455d520f80b36dc39562df079a3f29495e8f9447f"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "sed 's/gtk_icon_info_free/g_object_unref/' -i libindicator/indicator-image-helper.c",
+                        "sed 's/LIBS=\"-lm  $LIBS\"/LIBS=\" -lm  $LIBS\"/g' -i configure",
+                        "sed 's/LIBM=\"-lm\"/LIBM=\" -lm\"/g' -i configure"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "libappindicator",
+            "config-opts": [
+                "--with-gtk=3",
+                "--disable-gtk-doc-html",
+                "--disable-mono-test",
+                "--disable-static",
+                "--disable-tests"
+            ],
+            "rm-configure": true,
+            "sources": [{
+                    "type": "archive",
+                    "url": "https://launchpad.net/libappindicator/12.10/12.10.0/+download/libappindicator-12.10.0.tar.gz",
+                    "sha256": "d5907c1f98084acf28fd19593cb70672caa0ca1cf82d747ba6f4830d4cc3b49f"
+                },
+                {
+                    "type": "patch",
+                    "path": "no-python.patch"
```

Comment:
You can probably consider going more nuclear with this: https://github.com/flathub/io.github.Pithos/blob/master/appindicator-rip-build-system-apart.patch

Reply:
I'm going to use this patch I guess..

Reply:
Just took a look at the patch, the patch removes also Vala bindings...Not sure if that will affect using AppIndicator with this app?